__tests__/endpoints.test.js

```diff
@@ -40,4 +50,31 @@ describe('app endpoint tests', () => {
         });
     });
   });
+  describe('GET /api/articles', () => {
+    it('should respond with a status 200 and array of objects containing all articles data', () => {
+      return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles.length).toBe(12);
+
+          // testing individual properties on articles
+          articles.forEach((article) => {
+            expect(article).toHaveProperty('article_id', expect.any(Number));
```

Comment:
toMatchObject would work much better here. This matcher will still pass even if you add additional properties later on, so it makes for a nice flexible tests 👍